src/Nethermind/Nethermind.Core2/SszLimit.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Nethermind.Core2
+{
+    public static class SszLimit
+    {
+        public const ulong ValidatorRegistryLimit = 1_099_511_627_776;
```

Comment:
is it a result of some multiplication or some other meaningful number? would be nice to display as a power of something or the mult

Reply:
All these values actually need to come from the config files. They were hardcoded in the Nethermind SSZ, and I just moved from the data classes to a static class. For this pull request all I did is move the value.

The next PR (working on now) is moving across to the Nethermind SSZ, which will replace the hardcoded values with a Init function with the config values.